policy-controller/k8s/index/src/server.rs

```diff
@@ -178,6 +178,7 @@ impl SrvIndex {
 
     /// Update the index with a server instance.
     fn apply(&mut self, srv: policy::Server, ns_authzs: &AuthzIndex) {
+        trace!(?srv, "Applying server");
```

Comment:
nit/TIOLI: should this maybe have a span as well, so that the subsequent events have the `srv` field?

Reply:
the public entry points above include spans.